packages/adblocker/src/filters/network.ts

```diff
@@ -478,7 +555,17 @@ function getFilterReplaceOptionValue(
 
     if (code === 92 /* '\\' */) {
       parts[slashes] += line.slice(start, pos);
+      const [posAfterCharacterEscape, isCharacterEscape] = isCharacterEscapeInReplace(
+        line,
+        pos,
+        end,
+      );
+      if (isCharacterEscape === false) {
```

Comment:
That to verify: the intent here is to preserve only escape characters?

Reply:
It removes escaping character ('\\') by adding an offset to next `start` assignment.

Included in [`9b0d3ea` (#4241)](https://github.com/ghostery/adblocker/pull/4241/commits/9b0d3ea09f1671f1a0c5cf749d731fc5e92d67ad)